addons/binding/pom.xml

```diff
@@ -58,6 +58,7 @@
     <module>org.openhab.binding.knx</module>
     <module>org.openhab.binding.knx.test</module>
     <module>org.openhab.binding.kodi</module>
+	<module>org.openhab.binding.konnected</module>
```

Comment:
Indent with spaces.

Reply:
And please also add your binding to the [feature.xml file](https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml) in the alphabetical order.